modules/apps/site-initializer/site-initializer-raylife-ap/site-initializer-raylife-ap-test/src/testFunctional/macros/RaylifeAP.macro

```diff
@@ -20,6 +20,21 @@ definition {
 			locator1 = "RaylifeAP#MENU_SIDEBAR");
 	}
 
+	macro clickInApplicationKebab {
+		Click(locator1 = "RaylifeAP#APPLICATIONS_TABLE_KEBAB");
+	}
+
+	macro clickInApplicationKebabOption {
+		Click(
+			key_index = "${index}",
+			key_option = "${option}",
+			locator1 = "RaylifeAP#KEBAB_OPTION");
+	}
```

Comment:
Can we combine these two macros in only one? Why do we need to index here since we are giving the key_option?

Reply:
Yes, we can combine these two. The index is necessary because there are too many options with the same path, and the key_option because inside the kebab there are 2 options, that way we can specify the desired option.

Reply:
I made the changes